.github/workflows/lint.yml

```diff
@@ -53,10 +43,10 @@ jobs:
           sudo apt-get install -y -qq packer
       - name: Packer validate
         run: |
-          packer validate packer/ubuntu-build.json
-          packer validate packer/mint-build.json
+          packer validate .
           # Allow beta/test builds to fail validation
-          packer validate -var-file=packer/beta-vars.json packer/mint-build.json || true
+          packer validate -var-file=packer/mint-beta.pkrvars.hcl . || true
+        working-directory: packer
```

Comment:
How would you feel about adding a step like `packer fmt -diff -check packer/` to check formatting now that jsonlint is gone?

Reply:
Done. I used `working-directory` again so that it looks the same in both steps. And I reordered the commands because I don't remember if GH steps are effectively `set -e` and I don't think we should need to be able to remember :)